Helper/Config.php

```diff
@@ -654,6 +659,22 @@ public function getProductPageCheckoutFlag($store = null)
         );
     }
 
+    /**
+     * Get Order management flag from config
+     *
+     * @param int|string|Store $store
+     *
+     * @return  boolean
+     */
+    public function isOrderManagementEnabled($store = null)
```

Comment:
If we enable new feature via plugin settings and disable it by default, should we add feature switches as well?
+: We can disable remotely if necessary
-: It can be confusing for merchant staff: if we disable feature remotely they couldn't enable it via plugin settings.

@roopakv @matt-thomason @akulkapoor ,

Reply:
I think we should make a feature switch. We'll only disable it remotely in emergency cases - something is wrong on our end and we want to disable it without going through the CSM or requiring a plugin hotfix first.
@roopakv

Reply:
@matt-thomason @vitaliyreznikov since this is a new feature and probably not every merchant would want it enabled, especially the old ones, if there's no store side configuration then the feature switch should be OFF by default.

Reply:
@danac-gs We have store side configuration (in this PR) and it is disabled by default.

Reply:
@matt-thomason Added